Playground/GeneratePlayground.swift

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/swift
+import Foundation
+
+extension FileManager {
+    func directoryExists(at url: URL) -> Bool {
+        var isDirectory: ObjCBool = false
+        return fileExists(atPath: url.path, isDirectory: &isDirectory) && isDirectory.boolValue
+    }
+}
+
+let fileManager = FileManager()
```

Comment:
Can we use `FileManager.default`? From my understanding, initializing your own `FileManager` is only really useful if you're assigning the `delegate`.

```suggestion
let fileManager = FileManager.default
```